vispy/testing/_runners.py

```diff
@@ -190,8 +190,14 @@ def _check_line_endings():
 """
 
 
-def _examples():
-    """Run all examples and make sure they work
+def _examples(individial_file_paths):
+    """Run examples and make sure they work.
+
+    Parameters
+    ----------
+
+    individial_file_paths: str
```

Comment:
compromise - name it ```fname_string``` ? Since it makes it clear that it's supposed to be a string, which is then converted into a list by splitting it up?